### DISCUSSION TITLE
Testcase contribution for the library

### DISCUSSION THREAD

**Discussion by arindam-8** [jedisct1/libsodium#1273] [Ideas]
Hi,

I would like to contribute a few testcases for libsodium based on the library's usage as observed in its public clients. I was wondering if this would be of interest to the development team and if so, what would be the preferred way of going about doing this.

Thanks 
Arindam

**Comment by jedisct1** [comment]
Hi Arindam!

Of course, more test cases are always welcome!

And the preferred way would be to submit a pull request.

Just make sure that the tests work on all supported platforms, including JavaScript and WebAssembly.

**Comment by arindam-8** [comment]
Hi Frank,

Thanks a lot for the response! We are now in the process of integrating the testcases such that they fit right in with the existing framework. I will let you as soon as the PR is up and ready for review. Also, we'll make sure that the tests work on the supported platforms and thanks for pointing that out. 

Thanks
Arindam
